index.js

```diff
@@ -237,11 +237,10 @@ module.exports = input => {
 	}
 
 	if (check([0x0, 0x0, 0x0, 0x14, 0x66, 0x74, 0x79, 0x70, 0x71, 0x74, 0x20, 0x20]) ||
-		(buf[4] === 0x66 && buf[5] === 0x72 && buf[6] === 0x65 && buf[7] === 0x65) ||
-		((buf[4] === 0x66 && buf[5] === 0x74 && buf[6] === 0x79 && buf[7] === 0x70) &&
-		(buf[8] === 0x71 && buf[9] === 0x74 && buf[10] === 0x20 && buf[7] === 0x20)) ||
-		(buf[4] === 0x6D && buf[5] === 0x64 && buf[6] === 0x61 && buf[7] === 0x74) ||
-		(buf[4] === 0x77 && buf[5] === 0x69 && buf[6] === 0x64 && buf[7] === 0x65)) {
+		check([0x66, 0x72, 0x65, 0x65], {offset: 4}) ||
+		check([0x66, 0x74, 0x79, 0x70, 0x71, 0x74, 0x20, 0x20], {offset: 4}) ||
+		check([0x6D, 0x64, 0x61, 0x74], {offset: 4}) ||
+		check([0x77, 0x69, 0x64, 0x65], {offset: 4})) {
```

Comment:
Can you add an inline comment which of these are the `mjpeg` codec? For future reference.